addons/gamification/models/karma.py

```diff
@@ -2,9 +2,28 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 from werkzeug.exceptions import Forbidden
-from odoo import fields, models
+from odoo import api, fields, models
 
 
 class KarmaError(Forbidden):
     """ Karma-related error, used for forum and posts. """
     pass
+
+
+class KarmaRank(models.Model):
```

Comment:
Guidelining: rename to gamification_karma_rank . py (+ gamification_karma_rank_views.xml for views)